doc/Administration-guide/System-installation/Single-node-installation/index.md

```diff
@@ -622,40 +955,56 @@ Add the MongoDB repository to the /etc/apt/sources.list.d/mongodb.list.
 To install Kaa you will need to [download](http://www.kaaproject.org/download-kaa/) pre-built packages or [build](../Planning-your-deployment#building-kaa-from-source) them from the [source code](https://github.com/kaaproject/kaa). We will use pre-built packages in this guide.
 
 <ul class="nav nav-tabs">
-  <li class="active"><a data-toggle="tab" href="#Ubuntu1">Ubuntu 14.04</a></li>
-  <li><a data-toggle="tab" href="#CentOS1">CentOS 6.7</a></li>
+  <li class="active"><a data-toggle="tab" href="#Ubuntu16_">Ubuntu 16.04</a></li>
+  <li><a data-toggle="tab" href="#Ubuntu14_">Ubuntu 14.04</a></li>
```

Comment:
Here is no difference in installation instructions between two Ubuntu versions. Use only one tab for Ubuntu - 'Ubuntu 14.04/16.04'.